packages/sourcemapped/README.md

```diff
@@ -0,0 +1,59 @@
+# `sourcemapped`
+
+Helpers and utilities to work with source maps.
+
+### `fileToSource(contents: string, sourceName: string, sourcePath?: string): Promise<Source>`
+
+This function takes a `source`, a `sourceName` and an optional `sourcePath` and returns a source map-aware `Source` object. It will first try to load `${sourcePath}.map`, and if it is missing, it will try to load a source map from the given `content`.
```

Comment:
```suggestion
This function takes a `content`, a `sourceName` and an optional `sourcePath` and returns a source map-aware `Source` object. It will first try to load `${sourcePath}.map`, and if it is missing, it will try to load a source map from the given `content`.
```